frontend/src/components/layouts/FixedHeightLayout.js

```diff
@@ -19,13 +22,44 @@ const useStyles = makeStyles({
   },
 });
 
-export default function FixedHeightLayout({ title, children }) {
+export default function FixedHeightLayout({ 
+  title, 
+  children,
+  message,
+  messageType
+}) {
   const classes = useStyles();
-
+  const [initialMessageType, setInitialMessageType] = React.useState(null);
+  const [alertOpen, setAlertOpen] = React.useState(true);
+  const [initialMessage, setInitialMessage] = React.useState("");
+  useEffect(() => {
+    const params = getParams(window.location.href);
+    if (params.message) setInitialMessage(decodeURI(params.message));
+    if (params.errorMessage) {
```

Comment:
Can we guarantee that `params` will never be `undefined`? 

```suggestion
    if (params?.errorMessage) {
```

Reply:
Yeah, getParams always returns an object or an empty array. Therefore it's never undefined.